src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -419,7 +417,7 @@ public void ForceSomeOutOfDate()
                 DateTime time2 = File.GetLastWriteTime(t.OutputResources[1].ItemSpec);
 
                 System.Threading.Thread.Sleep(200);
-                if (NativeMethodsShared.IsOSX)
+                if (!NativeMethodsShared.IsWindows)
```

Comment:
> I'm very surprised to see this. It really should be only HFS+ that's in the dark ages of one-second granularity. Are you seeing otherwise?

The test was failing with expected true, actual false.  It doesn't print out the exact timing but I noticed the condition was OS X only and updating it made it pass on Linux.  

In my opinion, no test should ever depends on a a `Sleep`.  I can update the comment but I think my change makes sense.

> Are you seeing this on Linux? I had a similar fix earlier 0698cc9 . Are you seeing failures in TrackingDependenciesTests too?

The TrackingDependenciesTests are [disabled on .NET Core](https://github.com/Microsoft/msbuild/blob/xplat/src/Utilities/UnitTests/TrackedDependencies/TrackedDependenciesTests.cs#L20) and therefore don't run on Linux.